web/js/FlexyConnector.js

```diff
@@ -198,14 +198,14 @@ function validateWebID(id)
    return (id.length === 40);
 }
 
-//Validates an IP Address
+//Validates a URL
 //Must be in format xxx.xxx.xxx.xxx or xxx.xxx.xxx.xxx:pppp
-function validateIP(ip)
+function validateUrl(url)

```

Comment:
Shouldn't this method logic be updated? You are validating URLs (IP or DNS) against just regex for a valid IP address. This would fail for anything except a valid IP.

Reply:
My intent was to do this in another PR
https://github.com/hms-networks/flexy-osisoft-connector/issues/99